### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by 0xmohit**
- Reused contains() from utils.go at a couple of places
- Cleanup in return statements and boolean checks

### CODE REVIEW

File: server-rlimit-nix.go [hunk 1]
```diff
@@ -42,10 +42,7 @@ func setMaxOpenFiles() error {
 		return err
 	}
 	err = syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rLimit)
-	if err != nil {
-		return err
-	}
-	return nil
+	return err
```
Comment (harshavardhana) [inline server-rlimit-nix.go]: Further simplification is 

`return syscall.Getrlimit(syscal.RLIMIT_NOFILE, &rLimit)`

File: rpc-server.go [hunk 2]
```diff
@@ -47,10 +44,7 @@ func (s *storageServer) StatVolHandler(arg *string, reply *VolInfo) error {
 // DeleteVol operation.
 func (s *storageServer) DeleteVolHandler(arg *string, reply *GenericReply) error {
 	err := s.storage.DeleteVol(*arg)
-	if err != nil {
-		return err
-	}
-	return nil
```
Comment (harshavardhana) [inline rpc-server.go:53]: Further simplification is 
`return s.storage.DeleteVol(*arg)`

File: server-rlimit-nix.go (hunk 1 above)
Comment (0xmohit) [inline server-rlimit-nix.go]: Well, I wouldn't have probably invoked `syscall.Getrlimit` in this at all.

Comment (harshavardhana) [inline server-rlimit-nix.go]: The reason is to verify if Getrlimit is possible.

**Comment by harshavardhana** [comment]
Can you rebase this current master @0xmohit ?

**Comment by 0xmohit** [comment]
@harshavardhana Done.

**Comment by harshavardhana** [comment]
@0xmohit looks like all changes were reverted?

**Comment by harshavardhana** [comment]
Thanks @0xmohit for the patch merged it locally. Closing this PR.

**Pull request closed without merging**